src/pudl/workspace/datastore.py

```diff
@@ -175,8 +175,8 @@ class ZenodoDoiSettings(BaseSettings):
     # eia860m: ZenodoDoi = "10.5072/zenodo.1225517"
     eia861: ZenodoDoi = "10.5281/zenodo.10093091"
     # eia861: ZenodoDoi = "10.5072/zenodo.1229930"
-    eia923: ZenodoDoi = "10.5281/zenodo.8172818"
-    # eia923: ZenodoDoi = "10.5072/zenodo.1217724"
+    # eia923: ZenodoDoi = "10.5281/zenodo.8172818"
```

Comment:
We should remove all of the sandbox DOIs in here since Zenodo wiped the sandbox server during their backend migration, so all of them are now invalid.